docs/deployment/reference.md

```diff
@@ -111,9 +111,14 @@ parameters for the [rabbitMQ](https://github.com/bitnami/charts/tree/master/bitn
-| `dataLifecycle.image`                        | Default image for data lifecycle job                                                                                                                                                                               | `sslhep/servicex_minio_cleanup`                |
-| `dataLifecycle.tag`                          | Data lifecycle job image tag                                                                                                                                                                                       |                                                |
+| `dataLifecycle.image`                        | Default image for data lifecycle job                                                                                                                                                                               | `python`                                       |
+| `dataLifecycle.tag`                          | Data lifecycle job image tag                                                                                                                                                                                       | `3.10`                                         |
 | `dataLifecycle.pullPolicy`                   | Data lifecycle image pull policy                                                                                                                                                                                   | `Always`                                       |
 | `dataLifecycle.schedule`                     | Schedule for minioCleanup cronjob. See [reference](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#cron-schedule-syntax) for details on fields                                                | `* */8 * * *` (every 8 hours)                  |
 | `dataLifecycle.retention`                    | We will archive any transforms older than this. Use the gnu date command --date argument. See [date command](https://www.geeksforgeeks.org/date-command-linux-examples/#4-how-to-display-past-dates) for examples. | 7 days ago                                     |
 | `dataLifecycle.maxDesiredCacheSize`          | If the server-side cache is larger than this cleanup service will keep going forward in time to delete transforms. Specify units as Mb Gb, Tb or Pb                                                                | "1Tb"                                          |
+| `datasetLifecycle.image`                         | Default image for dataset cache lifecycle job                                                                                                                                                                  | `curlimages/curl`                              |
```

Comment:
I can't imagine a case where we would want a different image for initiating the dataset clean up as we use for the data lifecycle job.
In fact, I had imagined dataset cleanup to be a subset of data lifecycle, so ideally the schedule should just be a subitem of `dataLifecycle`

So, maybe 
dataLifecycle:
      image: curlimages/curl
      tag: "3.10"
      pullPolicy: Always
      enabled: true

    transformCleanup:
         schedule: "* */8 * * *"
         retention: "7 days ago"
         maxDesiredCacheSize: "1Tb"
    datasetCleanup:
        schedule: "0 * * * *"
        cacheLifetime: 24